staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -238,6 +239,20 @@ type Cloud struct {
 }
 
 func init() {
+	// In go-autorest SDK https://github.com/Azure/go-autorest/blob/master/autorest/sender.go#L258-L287,
+	// if ARM returns http.StatusTooManyRequests, the sender doesn't increase the retry attempt count,
```

Comment:
`doesn't decrease` ?

Reply:
`doesn't increase` means the request is counted as an attempt (it's a little odd within go-autorest logic).